### PR TITLE
Added Apollo examples for useSelect and useCombobox

### PR DESCRIPTION
**Pull request by ndeom**
Additional notes: had to install Apollo Client 3.0 to be able to use the new useQuery hook. React and React-DOM also had to be updated for this example

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,19 +5,20 @@
   "keywords": [],
   "main": "src/index.js",
   "dependencies": {
+    "@apollo/client": "3.1.1",
     "apollo-boost": "0.1.10",
     "axios": "0.18.0",
     "debounce-fn": "^3.0.1",
-    "downshift": "^3.1.8",
+    "downshift": "6.0.2",
```
Comment (kentcdodds) [inline package.json:12]: Could you verify the other examples still work with this update?

Comment (ndeom) [reply package.json:12]: Ah, yeah, ironically I forgot I updated that. Good catch! I went through each of the examples again and added/selected inputs to test each one. From what I can tell, none of the examples are throwing any errors with the newer version of Downshift installed.

Comment (kentcdodds) [reply package.json:12]: Thank you for taking the time!

Comment (ndeom) [reply package.json:12]: Of course! Just happy I could contribute. I'll keep an eye out for other issues. Also, this was issue [#997 ](https://github.com/downshift-js/downshift/issues/997) in the main repository, just fyi.

**Pull request merged**